pkgs/development/ocaml-modules/mustache/cli.nix

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
Should this be in by-name instead since it's not really a OCaml module?

Reply:
Yes, I have moved to `by-name`. Thank you for pointing out!